scripts/apache/get_fasta_sequence

```diff
@@ -29,49 +29,27 @@ sub get_fasta_sequence {
     my ($server) = @_;
 
     my $id = $server->require_argument('id');
+    my $seq_target = $server->require_argument('sequence_target');
 
-    my @raw_id_array = split (',', $id);
-    $id = "";
-    foreach my $current_id (@raw_id_array) {
-        if (index($current_id, '-') != -1) {
-          $current_id =~ s/\.\d+//;
-        }
-        $id = $id . ',' . $current_id;
-    }
```

Comment:
Multiply id's functionality  should not be removed, we expect any amount of id's

Reply:
While testing I noticed we get separate calls for each accession id from the list in the input field.

Reply:
So if we have that logic its going to be fetch for single accession id

Reply:
After changes we still should fetch multiply ids, all from the same source